cmd/run.go

```diff
@@ -594,7 +595,20 @@ func createBridgeRPC(
 
 func createRPC(cfg jRPC.Config, services []jRPC.Service) *jRPC.Server {
 	logger := log.WithFields("module", "RPC")
-	return jRPC.NewServer(cfg, services, jRPC.WithLogger(logger.GetSugaredLogger()))
+
+	healthHandler := healthcheck.NewHealthCheckHandler(logger)
+
+	defer func() {
```

Comment:
This has to be in the run functrion.

Reply:
Simpified logic and always returning a HTTP 200 (https://github.com/agglayer/aggkit/pull/264/commits/0af6b782250789aaa989a27e117781a0d6034b1b).